app/views/content/salaries-and-benefits.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: "Teacher salaries and pay scales"
-heading: "Salaries and benefits"
+title: "Teacher pay"
```

Comment:
So 'teacher pay' has an avg monthly search volume of 9,900, higher than 'teacher salary' which is 8,100. 'Teacher salary' does have a higher CTR/clicks, though it would be interesting to see why. Is it because the current title/meta is optimised for 'salary' and so uses the same language that users are searching for, making them more likely to click on it? If we changed to 'pay', would this lead to a higher CTR for that term, too? Looking at the SERPs, both terms seem to generate largely the same results.

I think looking at UR is hard because when we've done it in the past, the page has always been called 'salaries' so likely that users will repeat the same language back when talking about it?

The current page title (Teacher salaries and pay scales) doesn't have 'benefits' for space reasons, but if we have to drop 'pay scales' now, we can probs add it back in.

Think it could be an interesting test to see how CTR/clicks are affected, and could keep our eye on and change back if we see no added benefit in terms of clicks/CTR for that term. But equally happy to change back to salary! Thoughts? @gemmadallmandfe @philspray

Reply:
@EllieNodder I'd suggest adding `and benefits` back into the h1/title if we are going to try `Teacher pay` - we do talk (briefly!) about holidays and pensions, and they are both relatively good; it also feels a bit friendlier?